pipeline-examples/load-from-file/pipeline.groovy

```diff
@@ -0,0 +1,13 @@
+node {
```

Comment:
Because...*shrug*? No huge reason, more just that Jenkinsfiles are top-level and this isn't so my instincts said...